### PR TITLE
Added Sarai's recipe

### PR DESCRIPTION
**Pull request by svarona**
- [x] Create file for the recipe
- [x] Write the recipe
- [x] Translate the recipe

### CODE REVIEW

File: docs/drinks/beer.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Beer recipe?
+
+## English
+
+Beer is one of the oldest types of alcoholic drinks in the world, and the most widely consumed. It is the third most popular drink overall after potable water and tea. It is produced by the brewing and fermentation of starches, mainly derived from cereal grains—most commonly malted barley, though wheat, maize (corn), rice, and oats are also used. During the brewing process, fermentation of the starch sugars in the wort produces ethanol and carbonation in the resulting beer. Most modern beer is brewed with hops, which add bitterness and other flavours and act as a natural preservative and stabilising agent. Other flavouring agents such as gruit, herbs, or fruits may be included or used instead of hops. In commercial brewing, the natural carbonation effect is often removed during processing and replaced with forced carbonation.
```
Comment (amateo250) [inline docs/drinks/beer.md:5]: Vaya linea de mierda

File: docs/drinks/beer.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Svenska
+
+Öl är en dryck (i huvudsak med visst innehåll av alkohol) som har producerats i uppskattningsvis 7 000 år. Det första ölet skapades troligtvis då stärkelse- och sockerrika bröd blöttes och jäste i kontakt med vilda jästsvampar. Den moderna bryggeritekniken utvecklades i huvudsak i kloster under medeltiden och de största tekniska framstegen i industrialiserandet av bryggprocessen skedde under 1700- och 1800-talen. Bryggeriindustrin är idag en multinationell miljardindustri med Kina som största producentland.
```
Comment (amateo250) [inline docs/drinks/beer.md:13]: Excelente sueco

File: docs/drinks/beer.md (hunk 1 above)
Comment (svarona) [reply docs/drinks/beer.md:5]: No es de mierda

File: docs/drinks/beer.md (hunk 2 above)
Comment (svarona) [reply docs/drinks/beer.md:13]: Gracias!

**Review by sofstam — APPROVED** [review]

**Pull request merged**